system-helper/org.freedesktop.Flatpak.policy.in

```diff
@@ -127,6 +127,17 @@
     </defaults>
   </action>
 
+  <action id="org.freedesktop.Flatpak.update-remote">
+    <description>Update Remote</description>
+    <message>Authentication is required to update the configuration of software repositories</message>
+    <icon_name>package-x-generic</icon_name>
+    <defaults>
+      <allow_any>auth_admin</allow_any>
```

Comment:
I might be understanding polkit stuff wrong, but I think this describes the default rules for the policy, which we then override via the rules file to make it usable without any password via unprivileged users (my other patch to the rules file). But maybe I'm confusing things?